### DISCUSSION TITLE
player_id = -2

### DISCUSSION THREAD

**Discussion by ixu-repo** [deepmind/open_spiel#1142] [Q&A] [answered]
Hi,
I have that env = rl_environment.Environment(game="matrix_rps()")
When I run env.reset(), the current player_id is -2, which I think is not a valid player_id, I am wondering what might have caused this?

Thanks

**Comment by lanctot** [comment, marked as answer]
Ah yes, that is correct actually. -2 is a special player Id [referring to a simultaneous move node](https://github.com/google-deepmind/open_spiel/blob/03cf55f55daeab350e5b29ca240a0865b43fc789/open_spiel/spiel_globals.h#L30) (meaning multiple players act simultaneously), since this game is not turn-based.

BTW, for the repeated version of the game, you need to use the `repeated_game` game transform: https://github.com/google-deepmind/open_spiel/blob/03cf55f55daeab350e5b29ca240a0865b43fc789/open_spiel/python/examples/roshambo_population_example.py#L192
